### ISSUE TITLE
Fix Failing Rspec Tests

### ISSUE THREAD

**Issue by aaronskiba** [portagenetwork/roadmap#626]
Please complete the following fields as applicable:

**What version of the DMPRoadmap code are you running? (e.g. v2.2.0)**
- Upstream Release `v4.1.0`

**To-Do List**
- [x] #428
- [x] #627 
- [x] Ensure all GitHub Actions that execute Rspec tests complete successfully

**Comment by aaronskiba** [comment]
This issue is currently in progress within the following repo and branch: https://github.com/aaronskiba/roadmap/tree/aaron/merge-v4.1.0. The corresponding GitHub Action results are here: https://github.com/aaronskiba/roadmap/actions?query=branch%3Aaaron%2Fmerge-v4.1.0

**Comment by aaronskiba** [comment]
After adding `save_and_open_screenshot` to some of the tests in `./spec/features/plans/exports_spec.rb`, the following screenshot corresponds with some of the failures: 
![capybara-202401261456141489744026](https://github.com/portagenetwork/roadmap/assets/71047780/12fde869-db0b-458d-bfbb-4388db22155c)

And the following is outputted from the terminal:

```ruby
Failures:

  1) PlansExports User downloads their plan as CSV
     Failure/Error:
       within("#plan_#{plan.id}", visible: true) do
         click_button('Actions')
         click_link 'Download'
       end
     
     Capybara::ElementNotFound:
       Unable to find css "#plan_1"
     # /usr/share/rvm/gems/ruby-3.0.5@dmp/gems/capybara-3.39.2/lib/capybara/node/finders.rb:312:in `block in synced_resolve'
     # /usr/share/rvm/gems/ruby-3.0.5@dmp/gems/capybara-3.39.2/lib/capybara/node/base.rb:84:in `synchronize'
     # /usr/share/rvm/gems/ruby-3.0.5@dmp/gems/capybara-3.39.2/lib/capybara/node/finders.rb:301:in `synced_resolve'
     # /usr/share/rvm/gems/ruby-3.0.5@dmp/gems/capybara-3.39.2/lib/capybara/node/finders.rb:60:in `find'
     # /usr/share/rvm/gems/ruby-3.0.5@dmp/gems/capybara-3.39.2/lib/capybara/session.rb:773:in `find'
     # /usr/share/rvm/gems/ruby-3.0.5@dmp/gems/capybara-3.39.2/lib/capybara/session.rb:362:in `within'
     # /usr/share/rvm/gems/ruby-3.0.5@dmp/gems/capybara-3.39.2/lib/capybara/dsl.rb:52:in `call'
     # /usr/share/rvm/gems/ruby-3.0.5@dmp/gems/capybara-3.39.2/lib/capybara/dsl.rb:52:in `within_element'
     # /usr/share/rvm/gems/ruby-3.0.5@dmp/gems/capybara-3.39.2/lib/capybara/rspec/matcher_proxies.rb:15:in `within'
     # ./spec/features/plans/exports_spec.rb:133:in `block (2 levels) in <top (required)>'
     # /usr/share/rvm/gems/ruby-3.0.5@dmp/gems/webmock-3.19.1/lib/webmock/rspec.rb:39:in `block (2 levels) in <top (required)>'
```

**Comment by aaronskiba** [comment]
This is currently not an issue, and a fix for Upstream Release `v4.1.0` now exists within PR #700 that checks off everything within this issue's to-do list.
